### PR TITLE
[Utopia 380] fix protected route

### PR DESCRIPTION
**Pull request by binggaofw**
# Description

This PR includes the following proposed change(s):

- add protected route
- add a new 404 not found page

Screen Shot 2022-12-06 at 1 53 29 PM
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

**Comment by kushal-arora-fw** [comment]
@binggaofw  - when unauthenticated, there's a white blank space of our navbar we need to get rid of.

image

File: src/frontend/src/routes/router.tsx [hunk 1]
```diff
@@ -16,14 +29,17 @@ const Router = () => {
...
     <Routes>
-      <Route
-        path="/ppq"
-        element={<PPQLandingPage enablePiaIntakeForm={PIAIntakeFlag} />}
-      />
-      <Route path="/ppq-form" element={<PPQFormPage />} />
-      <Route path="/ppq-connect" element={<PPQConnectPage />} />
-      <Route path="/pia-intake" element={<PIAIntakeFormPage />} />
-      <Route path="/pia-result" element={<PIAIntakeResultsPage />} />
+      <Route path="*" element={<NotFound />} />
```
Comment (kushal-arora-fw) [inline src/frontend/src/routes/router.tsx:32]: Logically it makes sense to have NotFound route right at the end. Can we please move it to the bottom, if that does not affect the behaviour?

**Review by kushal-arora-fw — CHANGES_REQUESTED** [review]
Requesting changes for the blank space at the top when user is unauthenticated.

File: src/frontend/src/routes/router.tsx (hunk 1 above)
Comment (binggaofw) [reply src/frontend/src/routes/router.tsx:32]: will do it

**Comment by binggaofw** [comment]
@kushal-arora-fw update the css and remove the blank navbar space if user not login

**Review by kushal-arora-fw — APPROVED** [review]

**Pull request merged**